README.md

```diff
@@ -33,88 +33,6 @@ There are also [good first issues].
 [Maybe]: https://github.com/authzed/spicedb/labels/priority%2F4%20maybe
 [good first issues]: https://github.com/authzed/spicedb/labels/hint%2Fgood%20first%20issue
 
-## Getting Started
-
-### Installing the binary
```

Comment:
I could keep this here, but then we will have to remember to maintain two separate files